### ISSUE TITLE
Check Wazuh compatibility with new version AlmaLinux 9.3

### ISSUE THREAD

**Issue by jotacarma90** [wazuh/wazuh#20391]
# Description
Hello team, this issue is to check the full compatibility of Wazuh on the newfound version of AlmaLinux 9.3 operating system.

OSs checks issue: https://github.com/wazuh/wazuh/issues/20373

For this, it is necessary to perform the following tests to check that everything works as expected:

- [x] Agent and server (if possible) instalations.
- [x] Enrollment and connectivity with the manager.
- [x] WPK  upgrade.
- [x] FIM: Realtime and whodata engines (if available).
- [x] SCA: Policy support.
- [x] Vulnerability Detector: Vulnerability support.
- [x] Syscollector: Complete inventory.
- [x] Active Response.
- [x] Log capture (Eventchannel for Windows and ULS for macOS).

**Comment by ncvicchi** [comment]
# Testing

###   :green_circle:  Agent and server, (enrollment and connectivity with the manager)

Agent installed at the AlmaLinux machine by following [this guide](https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh-agent-package-linux.html).
The manager was implemented by running an the OVA 4.7.0 release.

<details><summary>Agent log</summary>

```console

2023/11/29 16:48:19 wazuh-execd: INFO: Started (pid: 51496).
2023/11/29 16:48:20 wazuh-agentd: INFO: (1410): Reading authentication keys file.
2023/11/29 16:48:20 wazuh-agentd: INFO: Using notify time: 10 and max time to reconnect: 60
2023/11/29 16:48:20 wazuh-agentd: INFO: Version detected -> Linux |alma93 |5.14.0-362.8.1.el9_3.x86_64 |#1 SMP PREEMPT_DYNAMIC Tue Nov 7 14:54:22 EST 2023 |x86_64 [AlmaLinux|almalinux: 9.3 (Shamrock Pampas Cat)] - Wazuh v4.7.0
2023/11/29 16:48:20 wazuh-agentd: INFO: Started (pid: 51518).
2023/11/29 16:48:20 wazuh-agentd: INFO: Requesting a key from server: 192.168.0.152
2023/11/29 16:48:20 wazuh-agentd: INFO: No authentication password provided
2023/11/29 16:48:20 wazuh-agentd: INFO: Using agent name as: alma93
2023/11/29 16:48:20 wazuh-agentd: INFO: Waiting for server reply
2023/11/29 16:48:20 wazuh-agentd: INFO: Valid key received
2023/11/29 16:48:20 wazuh-agentd: INFO: Waiting 20 seconds before server connection
2023/11/29 16:48:40 wazuh-agentd: INFO: (1410): Reading authentication keys file.
2023/11/29 16:48:40 wazuh-agentd: INFO: Using AES as encryption method.
2023/11/29 16:48:40 wazuh-agentd: INFO: Trying to connect to server ([192.168.0.152]:1514/tcp).
2023/11/29 16:48:40 wazuh-agentd: INFO: (4102): Connected to the server ([192.168.0.152]:1514/tcp).
2023/11/29 16:48:40 wazuh-agentd: INFO: Agent is restarting due to shared configuration changes.
2023/11/29 16:48:44 wazuh-agentd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2023/11/29 16:48:44 wazuh-execd: INFO: (1314): Shutdown received. Deleting responses.
2023/11/29 16:48:44 wazuh-execd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2023/11/29 16:48:45 wazuh-execd: INFO: Started (pid: 51886).
2023/11/29 16:48:45 wazuh-agentd: INFO: (1410): Reading authentication keys file.
2023/11/29 16:48:45 wazuh-agentd: INFO: Using notify time: 10 and max time to reconnect: 60
2023/11/29 16:48:45 wazuh-agentd: INFO: Version detected -> Linux |alma93 |5.14.0-362.8.1.el9_3.x86_64 |#1 SMP PREEMPT_DYNAMIC Tue Nov 7 14:54:22 EST 2023 |x86_64 [AlmaLinux|almalinux: 9.3 (Shamrock Pampas Cat)] - Wazuh v4.7.0
2023/11/29 16:48:45 wazuh-agentd: INFO: Started (pid: 51895).
2023/11/29 16:48:45 wazuh-agentd: INFO: Using AES as encryption method.
2023/11/29 16:48:45 wazuh-agentd: INFO: Trying to connect to server ([192.168.0.152]:1514/tcp).
2023/11/29 16:48:45 wazuh-agentd: INFO: (4102): Connected to the server ([192.168.0.152]:1514/tcp).


```

</details>

<details><summary>wazuh-control status before reboot</summary>

```console

[root@alma93 wazuh]# /var/ossec/bin/wazuh-control status
wazuh-modulesd is running...
wazuh-logcollector is running...
wazuh-syscheckd is running...
wazuh-agentd is running...
wazuh-execd is running...
[root@alma93 wazuh]#


```
</details>

<details><summary>wazuh-control status after reboot</summary>

```console

[root@alma93 wazuh]# /var/ossec/bin/wazuh-control status
wazuh-modulesd is running...
wazuh-logcollector is running...
wazuh-syscheckd is running...
wazuh-agentd is running...
wazuh-execd is running...
[root@alma93 wazuh]#

```

</details>

<details><summary>Manager log</summary>

```

2023/11/29 21:48:20 wazuh-authd: INFO: New connection from 192.168.0.204
2023/11/29 21:48:20 wazuh-authd: INFO: Received request for a new agent (alma93) from: 192.168.0.204
2023/11/29 21:48:20 wazuh-authd: INFO: Agent key generated for 'alma93' (requested by any)
2023/11/29 21:48:22 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
2023/11/29 21:48:22 wazuh-remoted: INFO: (1410): Reading authentication keys file.

```
</details>

<details><summary>Dashboard</summary>

![image](https://github.com/wazuh/wazuh/assets/69121070/a04d6ba0-481d-47c1-8c97-128e1196fae8)

</details>

###  :green_circle: FIM scheduled

Using default configuration (just changed frequency to 60 seconds):

<details><summary>Results</summary>

```console
[root@alma93 wazuh]# touch /etc/test1.txt
[root@alma93 wazuh]# nano /etc/test1.txt
[root@alma93 wazuh]# rm /etc/test1.txt
rm: remove regular file '/etc/test1.txt'? y
```

```console

** Alert 1701296421.613538: - ossec,syscheck,syscheck_entry_added,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:20:21 (alma93) any->syscheck
Rule: 554 (level 5) -> 'File added to the system.'
File '/etc/test1.txt' added
Mode: scheduled

Attributes:
 - Size: 0
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:19:30 2023
 - Inode: 35292029
 - User: root (0)
 - Group: root (0)
 - MD5: d41d8cd98f00b204e9800998ecf8427e
 - SHA1: da39a3ee5e6b4b0d3255bfef95601890afd80709
 - SHA256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855

** Alert 1701296484.614217: - ossec,syscheck,syscheck_entry_modified,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:21:24 (alma93) any->syscheck
Rule: 550 (level 7) -> 'Integrity checksum changed.'
File '/etc/test1.txt' modified
Mode: scheduled
Changed attributes: size,mtime,md5,sha1,sha256
Size changed from '0' to '5'
Old modification time was: '1701296370', now it is '1701296465'
Old md5sum was: 'd41d8cd98f00b204e9800998ecf8427e'
New md5sum is : 'd8e8fca2dc0f896fd7cb4cb0031ba249'
Old sha1sum was: 'da39a3ee5e6b4b0d3255bfef95601890afd80709'
New sha1sum is : '4e1243bd22c66e76c2ba9eddc1f91394e57f9f83'
Old sha256sum was: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
New sha256sum is : 'f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2'

Attributes:
 - Size: 5
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:21:05 2023
 - Inode: 35292029
 - User: root (0)
 - Group: root (0)
 - MD5: d8e8fca2dc0f896fd7cb4cb0031ba249
 - SHA1: 4e1243bd22c66e76c2ba9eddc1f91394e57f9f83
 - SHA256: f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2

** Alert 1701296673.615438: - ossec,syscheck,syscheck_entry_deleted,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:24:33 (alma93) any->syscheck
Rule: 553 (level 7) -> 'File deleted.'
File '/etc/test1.txt' deleted
Mode: scheduled

Attributes:
 - Size: 5
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:21:05 2023
 - Inode: 35292029
 - User: root (0)
 - Group: root (0)
 - MD5: d8e8fca2dc0f896fd7cb4cb0031ba249
 - SHA1: 4e1243bd22c66e76c2ba9eddc1f91394e57f9f83
 - SHA256: f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2


```
  
</details>

###  :green_circle: FIM: Real-time

Configuration:
```
    <directories realtime="yes">/testrealtime</directories>
```

<details><summary>Results</summary>

```console
[root@alma93 testrealtime]# touch test1.txt
[root@alma93 testrealtime]# nano test1.txt
[root@alma93 testrealtime]# rm test1.txt
rm: remove regular file 'test1.txt'? y

```

```console

** Alert 1701297053.629924: - ossec,syscheck,syscheck_entry_added,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:30:53 (alma93) any->syscheck
Rule: 554 (level 5) -> 'File added to the system.'
File '/testrealtime/test1.txt' added
Mode: realtime

Attributes:
 - Size: 0
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:30:53 2023
 - Inode: 17735715
 - User: root (0)
 - Group: root (0)
 - MD5: d41d8cd98f00b204e9800998ecf8427e
 - SHA1: da39a3ee5e6b4b0d3255bfef95601890afd80709
 - SHA256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855

** Alert 1701297063.630611: - ossec,syscheck,syscheck_entry_modified,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:31:03 (alma93) any->syscheck
Rule: 550 (level 7) -> 'Integrity checksum changed.'
File '/testrealtime/test1.txt' modified
Mode: realtime
Changed attributes: size,mtime,md5,sha1,sha256
Size changed from '0' to '5'
Old modification time was: '1701297053', now it is '1701297063'
Old md5sum was: 'd41d8cd98f00b204e9800998ecf8427e'
New md5sum is : 'd8e8fca2dc0f896fd7cb4cb0031ba249'
Old sha1sum was: 'da39a3ee5e6b4b0d3255bfef95601890afd80709'
New sha1sum is : '4e1243bd22c66e76c2ba9eddc1f91394e57f9f83'
Old sha256sum was: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
New sha256sum is : 'f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2'

Attributes:
 - Size: 5
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:31:03 2023
 - Inode: 17735715
 - User: root (0)
 - Group: root (0)
 - MD5: d8e8fca2dc0f896fd7cb4cb0031ba249
 - SHA1: 4e1243bd22c66e76c2ba9eddc1f91394e57f9f83
 - SHA256: f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2

** Alert 1701297068.631840: - ossec,syscheck,syscheck_entry_deleted,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:31:08 (alma93) any->syscheck
Rule: 553 (level 7) -> 'File deleted.'
File '/testrealtime/test1.txt' deleted
Mode: realtime

Attributes:
 - Size: 5
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:31:03 2023
 - Inode: 17735715
 - User: root (0)
 - Group: root (0)
 - MD5: d8e8fca2dc0f896fd7cb4cb0031ba249
 - SHA1: 4e1243bd22c66e76c2ba9eddc1f91394e57f9f83
 - SHA256: f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2


```

</details>

###  :green_circle: FIM whodata

Configuration:
```
    <directories whodata="yes">/testwhodata</directories>
```

<details><summary>Results</summary>

```console
[root@alma93 testwhodata]# touch test1.txt
[root@alma93 testwhodata]# nano test1.txt
[root@alma93 testwhodata]# rm test1.txt
rm: remove regular file 'test1.txt'? y
[root@alma93 testwhodata]#

```

```console

** Alert 1701297156.632519: - ossec,syscheck,syscheck_entry_added,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:32:36 (alma93) any->syscheck
Rule: 554 (level 5) -> 'File added to the system.'
File '/testwhodata/test1.txt' added
Mode: whodata

Attributes:
 - Size: 0
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:32:36 2023
 - Inode: 35301837
 - User: root (0)
 - Group: root (0)
 - MD5: d41d8cd98f00b204e9800998ecf8427e
 - SHA1: da39a3ee5e6b4b0d3255bfef95601890afd80709
 - SHA256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
 - (Audit) User name: root
 - (Audit) Audit name: wazuh
 - (Audit) Effective name: root
 - (Audit) Group name: root
 - (Audit) Process id: 21416
 - (Audit) Process name: /usr/bin/touch
 - (Audit) Process cwd: /testwhodata
 - (Audit) Parent process name: /usr/bin/bash
 - (Audit) Parent process id: 3254
 - (Audit) Parent process cwd: /testwhodata

** Alert 1701297165.633551: - ossec,syscheck,syscheck_entry_modified,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:32:45 (alma93) any->syscheck
Rule: 550 (level 7) -> 'Integrity checksum changed.'
File '/testwhodata/test1.txt' modified
Mode: whodata
Changed attributes: size,mtime,md5,sha1,sha256
Size changed from '0' to '5'
Old modification time was: '1701297156', now it is '1701297165'
Old md5sum was: 'd41d8cd98f00b204e9800998ecf8427e'
New md5sum is : 'd8e8fca2dc0f896fd7cb4cb0031ba249'
Old sha1sum was: 'da39a3ee5e6b4b0d3255bfef95601890afd80709'
New sha1sum is : '4e1243bd22c66e76c2ba9eddc1f91394e57f9f83'
Old sha256sum was: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
New sha256sum is : 'f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2'

Attributes:
 - Size: 5
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:32:45 2023
 - Inode: 35301837
 - User: root (0)
 - Group: root (0)
 - MD5: d8e8fca2dc0f896fd7cb4cb0031ba249
 - SHA1: 4e1243bd22c66e76c2ba9eddc1f91394e57f9f83
 - SHA256: f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2
 - (Audit) User name: root
 - (Audit) Audit name: wazuh
 - (Audit) Effective name: root
 - (Audit) Group name: root
 - (Audit) Process id: 21461
 - (Audit) Process name: /usr/bin/nano
 - (Audit) Process cwd: /testwhodata
 - (Audit) Parent process name: /usr/bin/bash
 - (Audit) Parent process id: 3254
 - (Audit) Parent process cwd: /testwhodata

** Alert 1701297174.635124: - ossec,syscheck,syscheck_entry_deleted,syscheck_file,pci_dss_11.5,gpg13_4.11,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,
2023 Nov 29 22:32:54 (alma93) any->syscheck
Rule: 553 (level 7) -> 'File deleted.'
File '/testwhodata/test1.txt' deleted
Mode: whodata

Attributes:
 - Size: 5
 - Permissions: rw-r--r--
 - Date: Wed Nov 29 22:32:45 2023
 - Inode: 35301837
 - User: root (0)
 - Group: root (0)
 - MD5: d8e8fca2dc0f896fd7cb4cb0031ba249
 - SHA1: 4e1243bd22c66e76c2ba9eddc1f91394e57f9f83
 - SHA256: f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2
 - (Audit) User name: root
 - (Audit) Audit name: wazuh
 - (Audit) Effective name: root
 - (Audit) Group name: root
 - (Audit) Process id: 21506
 - (Audit) Process name: /usr/bin/rm
 - (Audit) Process cwd: /testwhodata
 - (Audit) Parent process name: /usr/bin/bash
 - (Audit) Parent process id: 3254
 - (Audit) Parent process cwd: /testwhodata


```

</details>

### :yellow_circle: SCA support

No official SCA policies are available ( [check](https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/available-sca-policies.html) this list).

Since it is not listed at [CIS Benchmarks](https://downloads.cisecurity.org/#/), no issue will be created.

SCA module is tested anyway, with a custom policy.

Configuration

[This usecase](https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/use-cases.html#detecting-keyword-in-a-file) is used to test SCA module.

<details><summary>Agent log</summary>

```console

2023/11/29 17:39:03 sca: INFO: Module started.
2023/11/29 17:39:03 wazuh-modulesd:ciscat: INFO: Module disabled. Exiting...
2023/11/29 17:39:03 sca: INFO: Loaded policy '/var/ossec/ruleset/sca/cis_rhel9_linux.yml'
2023/11/29 17:39:03 sca: INFO: Loaded policy '/var/ossec/etc/custom-sca-files/keywordcheck.yml'
2023/11/29 17:39:03 sca: INFO: Starting Security Configuration Assessment scan.
2023/11/29 17:39:03 wazuh-modulesd:syscollector: INFO: Module started.
2023/11/29 17:39:03 wazuh-modulesd:syscollector: INFO: Starting evaluation.
2023/11/29 17:39:03 sca: INFO: Skipping policy '/var/ossec/ruleset/sca/cis_rhel9_linux.yml': 'Check RHEL9 family platform.'
2023/11/29 17:39:03 sca: INFO: Starting evaluation of policy: '/var/ossec/etc/custom-sca-files/keywordcheck.yml'
2023/11/29 17:39:04 wazuh-modulesd:syscollector: INFO: Evaluation finished.
2023/11/29 17:39:04 wazuh-syscheckd: INFO: (6000): Starting daemon...
2023/11/29 17:39:04 wazuh-syscheckd: INFO: (6010): File integrity monitoring scan frequency: 60 seconds
2023/11/29 17:39:04 wazuh-syscheckd: INFO: (6008): File integrity monitoring scan started.
2023/11/29 17:39:04 rootcheck: INFO: Starting rootcheck scan.
2023/11/29 17:39:06 wazuh-syscheckd: INFO: (6009): File integrity monitoring scan ended.
2023/11/29 17:39:06 wazuh-syscheckd: INFO: FIM sync module started.
2023/11/29 17:39:06 wazuh-syscheckd: INFO: (6019): File integrity monitoring real-time Whodata engine started.
2023/11/29 17:39:06 wazuh-syscheckd: INFO: (6012): Real-time file integrity monitoring started.
2023/11/29 17:39:06 sca: INFO: Evaluation finished for policy '/var/ossec/etc/custom-sca-files/keywordcheck.yml'
2023/11/29 17:39:06 sca: INFO: Security Configuration Assessment scan finished. Duration: 3 seconds.

```

</details>

<details><summary>Manager log</summary>

```console

** Alert 1701297548.639012: - sca,gdpr_IV_35.7.d,pci_dss_2.2,nist_800_53_CM.1,tsc_CC7.1,tsc_CC7.2,
2023 Nov 29 22:39:08 (alma93) any->sca
Rule: 19007 (level 7) -> 'SCA use case: Keyword check: Ensure password is disabled in the test configuration file'
{"type":"check","id":1170049407,"policy":"SCA use case: Keyword check","policy_id":"keyword_check","check":{"id":10000,"title":"Ensure password is disabled in the test configuration file","description":"Password is enabled in the test configuration file.","rationale":"Password is considered weak for the custom test application. Threat actors can brute-force your password.","remediation":"Disable password by setting the value of the password_enabled option to no.","rules":["f:/usr/share/testfile.txt -> r:^password_enabled: yes$"],"condition":"none","file":"/usr/share/testfile.txt","result":"failed"}}
sca.type: check
sca.scan_id: 1170049407
sca.policy: SCA use case: Keyword check
sca.check.id: 10000
sca.check.title: Ensure password is disabled in the test configuration file
sca.check.description: Password is enabled in the test configuration file.
sca.check.rationale: Password is considered weak for the custom test application. Threat actors can brute-force your password.
sca.check.remediation: Disable password by setting the value of the password_enabled option to no.
sca.check.file: ["/usr/share/testfile.txt"]
sca.check.result: failed

** Alert 1701297555.640419: - sca,gdpr_IV_35.7.d,pci_dss_2.2,nist_800_53_CM.1,tsc_CC7.1,tsc_CC7.2,
2023 Nov 29 22:39:15 (alma93) any->sca
Rule: 19005 (level 9) -> 'SCA summary: SCA use case: Keyword check: Score less than 30% (0)'
{"type":"summary","scan_id":1170049407,"name":"SCA use case: Keyword check","policy_id":"keyword_check","file":"keywordcheck.yml","description":"Guidance for checking for a keyword or phrase in files on Ubuntu endpoints.","references":"https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/index.html,https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/creating-custom-policies.html","passed":0,"failed":1,"invalid":0,"total_checks":1,"score":0,"start_time":1701297543,"end_time":1701297543,"hash":"5d28a90f4498a81461efbaf6f628a19d9778390bb5c81a393dd936181cc3d826","hash_file":"a33741eefba60ba75052477564b607cc97bdf36c773194c7ead40f30e356ec4d","force_alert":"1"}
sca.type: summary
sca.scan_id: 1170049407
sca.policy: SCA use case: Keyword check
sca.description: Guidance for checking for a keyword or phrase in files on Ubuntu endpoints.
sca.policy_id: keyword_check
sca.passed: 0
sca.failed: 1
sca.invalid: 0
sca.total_checks: 1
sca.score: 0
sca.file: keywordcheck.yml

** Alert 1701297564.641689: - sca,gdpr_IV_35.7.d,pci_dss_2.2,nist_800_53_CM.1,tsc_CC7.1,tsc_CC7.2,
2023 Nov 29 22:39:24 (alma93) any->sca
Rule: 19005 (level 9) -> 'SCA summary: SCA use case: Keyword check: Score less than 30% (0)'
{"type":"summary","scan_id":1170049407,"name":"SCA use case: Keyword check","policy_id":"keyword_check","file":"keywordcheck.yml","description":"Guidance for checking for a keyword or phrase in files on Ubuntu endpoints.","references":"https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/index.html,https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/creating-custom-policies.html","passed":0,"failed":1,"invalid":0,"total_checks":1,"score":0,"start_time":1701297543,"end_time":1701297543,"hash":"5d28a90f4498a81461efbaf6f628a19d9778390bb5c81a393dd936181cc3d826","hash_file":"a33741eefba60ba75052477564b607cc97bdf36c773194c7ead40f30e356ec4d","force_alert":"1","force_alert":"1"}
sca.type: summary
sca.scan_id: 1170049407
sca.policy: SCA use case: Keyword check
sca.description: Guidance for checking for a keyword or phrase in files on Ubuntu endpoints.
sca.policy_id: keyword_check
sca.passed: 0
sca.failed: 1
sca.invalid: 0
sca.total_checks: 1
sca.score: 0
sca.file: keywordcheck.yml


```

</details>

<details><summary>Dashboard</summary>

![Image](https://github.com/wazuh/wazuh/assets/69121070/11e70ef9-93d0-4931-a86e-82b8e9160679)

![Image](https://github.com/wazuh/wazuh/assets/69121070/8f80b07b-b579-4180-be5b-00a30637bc4e)

</details>

### :green_circle: Syscollector

Configuration by default

<details><summary>Agent log</summary>

```

2023/11/29 17:39:03 wazuh-modulesd:syscollector: INFO: Module started.
2023/11/29 17:39:03 wazuh-modulesd:syscollector: INFO: Starting evaluation.
2023/11/29 17:39:04 wazuh-modulesd:syscollector: INFO: Evaluation finished.

```

</details>


<details><summary>Inventory</summary>

![Image](https://github.com/wazuh/wazuh/assets/69121070/f4b7f276-dd5a-4303-9adc-23d117753a3f)

![Image](https://github.com/wazuh/wazuh/assets/69121070/e2c6780f-4183-49a7-8dbc-8892d4f8b77e)

![Image](https://github.com/wazuh/wazuh/assets/69121070/19fe4d0c-fb36-450d-87e1-834b241f225a)

</details>

###  :green_circle: Vulnerability detector

AlmaLinux 9.3 is officially supported in [Version 4.7.0](https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/how-it-works.html#vuln-det-compatibility-matrix)

Both vulnerability detection module as well as AlmaLinux vulnerability detection were enabled at the manager's ossec.log

```console
<vulnerability-detector>
  <enabled>yes</enabled>
  <interval>5m</interval>
  <min_full_scan_interval>6h</min_full_scan_interval>
  <run_on_start>yes</run_on_start>
...
<!-- Alma Linux OS vulnerabilities -->
<provider name="almalinux">
  <enabled>yes</enabled>
  <os>8</os>
  <os>9</os>
  <update_interval>1h</update_interval>
</provider>

```

<details><summary>Results</summary>

```console

[root@wazuh-server alerts]# grep  wazuh-modulesd:vulnerability-detector: /var/ossec/logs/ossec.log
2023/11/29 22:53:43 wazuh-modulesd:vulnerability-detector: INFO: (5400): Starting 'AlmaLinux 8' database update.
2023/11/29 22:53:53 wazuh-modulesd:vulnerability-detector: INFO: (5430): The update of the 'AlmaLinux 8' feed finished successfully.
2023/11/29 22:53:53 wazuh-modulesd:vulnerability-detector: INFO: (5400): Starting 'AlmaLinux 9' database update.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5430): The update of the 'AlmaLinux 9' feed finished successfully.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5400): Starting 'National Vulnerability Database' database update.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5430): The update of the 'National Vulnerability Database' feed finished successfully.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5400): Starting 'Microsoft Security Update' database update.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5430): The update of the 'Microsoft Security Update' feed finished successfully.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5431): Starting vulnerability scan.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: WARNING: (5575): Unavailable vulnerability data for the agent '000' OS. Skipping it.
2023/11/29 22:53:57 wazuh-modulesd:vulnerability-detector: INFO: (5450): Analyzing agent '002' vulnerabilities.
2023/11/29 22:54:08 wazuh-modulesd:vulnerability-detector: INFO: (5471): Finished vulnerability assessment for agent '002'
2023/11/29 22:54:08 wazuh-modulesd:vulnerability-detector: INFO: (5472): Vulnerability scan finished.


```

![Image](https://github.com/wazuh/wazuh/assets/69121070/bac20715-d0dc-4865-bea3-9bcccca02e3e)

</details>


### 🟢  Upgrade using WPK

WPK was downloaded from [here](https://documentation.wazuh.com/current/user-manual/agents/remote-upgrading/wpk-list.html) and [this ](https://documentation.wazuh.com/current/user-manual/agents/remote-upgrading/install-custom-wpk.html#run-the-upgrade) procedure was follow (from step 2)

<details><summary>Results</summary>

```console

[root@wazuh-server alerts]# /var/ossec/bin/agent_upgrade -a 002 -f /home/wazuh-user/wazuh_agent_v4.7.0_linux_x86_64.wpk -x upgrade.sh

Upgrading...

Upgraded agents:
        Agent 002 upgraded: Wazuh v4.7.0 -> Wazuh v4.7.0
[root@wazuh-server alerts]#

```

</details>

**Comment by lchico** [comment]
LGTM!
